src/main/flight/mixer.c

```diff
@@ -396,6 +398,8 @@ void mixTable(const float dT)
                 } else {
                     motor[i] = motorConfig()->minthrottle;
                 }
+            } else {
+                motor[i] = linLookupThrottle(motor[i]);
```

Comment:
Using lookup table is not worth the effort. With available CPU power and availability of FPU we should apply the formula directly.